Document/CategoryManager.php

```diff
@@ -22,7 +22,7 @@ class CategoryManager extends BaseDocumentManager implements CategoryManagerInte
     /**
      * {@inheritdoc}
      */
-    public function getPager(array $criteria, $page, $maxPerPage = 10)
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
```

Comment:
@Soullivaneuh I have this bug when Symfony cache is warming. I don't know when it appears because I worked on m'y actual project since few days.